analysis_options.yaml

```diff
@@ -1,77 +1,75 @@
 # Specify analysis options.
 #
 # This file is a copy of analysis_options.yaml from flutter repo
-# as of 2021-03-19, but with:
-#  - "always_require_non_null_named_parameters" disabled (because we
-#    can't import the meta package from the SDK),
-#  - "avoid_equals_and_hash_code_on_mutable_classes" disabled (same reason),
-#  - "missing_whitespace_between_adjacent_strings" disabled (too many false
-#     positives),
-#  - "sort_constructors_first" disabled (because we have private fake
-#     constructors),
-#  - "prefer_final_fields" disabled (because we do weird things with
-#    private fields, especially on the PlatformDispatcher object),
-#  - "public_member_api_docs" enabled.
+# as of 2022-07-27, but with some modifications marked with
+# "DIFFERENT FROM FLUTTER/FLUTTER" below.
 
 analyzer:
   exclude:
     # Fixture depends on dart:ui and raises false positives.
     - flutter_frontend_server/test/fixtures/lib/main.dart
+  language:
+    strict-casts: true
+    strict-raw-types: true
   errors:
     # treat missing required parameters as a warning (not a hint)
     missing_required_param: warning
     # treat missing returns as a warning (not a hint)
     missing_return: warning
-    native_function_body_in_non_sdk_code: ignore
-    # allow having TODOs in the code
+    native_function_body_in_non_sdk_code: ignore # DIFFERENT FROM FLUTTER/FLUTTER
```

Comment:
We shouldn't need this anymore now that we're on FfiNative.

Reply:
Mind if I remove this in a follow-up?

Reply:
Trying to turn this on in https://github.com/flutter/engine/pull/35015 to see what (if anything) breaks.